setup.py

```diff
@@ -27,8 +27,8 @@
 ase_deps = ["ase~=3.20"]
 cif_deps = ["numpy~=1.19"]
 pdb_deps = cif_deps
-pymatgen_deps = ["pymatgen==2020.9.14"]
-jarvis_deps = ["jarvis-tools==2020.10.3"]
+pymatgen_deps = ["pymatgen==2020.10.9"]
```

Comment:
```suggestion
pymatgen_deps = ["pymatgen==2020.10.9.1"]
```

Reply:
I tried this first, but setuptools didn't like patch versions in this format in the CI. Did you try it locally?

Reply:
Ah, the problem isn't with setuptools, instead its with this version regexp in the tests:

https://github.com/Materials-Consortia/optimade-python-tools/blob/004686c705bf76d0dd228ba84b9a91a7d62d53a7/tests/adapters/structures/utils.py#L11

Reply:
Have tweaked it and the version number, lets see if `importorskip` works with this extra number

Reply:
Seems to be good now!